welcome.md

```diff
@@ -0,0 +1,37 @@
+---
+
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+
+---
+
+UMA stands for "Universal Market Access" 
+
+Our mission is to make financial markets universally fair and accessible.
+
+UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a pre-requisite to a free, open and fair financial system. 
+
+UMA's goal is to enable anyone to attain or transfer any form of risk seamlessly and securely through the UMA protocol thus empowering everyone to participate in a universally accessible financial system.
+
+## What is UMA ##
+
+UMA builds open source infrastructure for developers to create financial instruments in a permissionless, secure and efficient manner. 

```

Comment:
Should we call out synthetic assets here versus financial instruments? Or say how the first area UMA is focusing on is synthetic assets?